driver/pool_driver.hpp

```diff
@@ -271,6 +271,7 @@ int PoolDriver_impl<Tgpu, Tref, Index>::SetPoolDescriptorFromCmdLineArgs()
     }
     else
     {
+        std::cout << inflags.GetValueStr("pad_mode") << std::endl;
```

Comment:
Are there any specific reasons you do this?

Reply:
No, this is related to another ticket that I forgot to delete here. Removed it.